gc/scheduler/scheduler.go

```diff
@@ -306,8 +309,8 @@ func (s *gcScheduler) run(ctx context.Context) {
 		}
 
 		s.waiterL.Lock()
-
-		stats, err := s.c.GarbageCollect(ctx)
+		tracCtx, startSpan := tracing.StartSpan(ctx, "StartGarbageCollect")
```

Comment:
IMO we should probably follow a common naming convention for span names atleast across containerd. I could not find any specific semantic convention around span names, the only recommendation available is [here](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md), which basically says that the span names should not be  generic nor  too specific. 

In my [PR](https://github.com/containerd/containerd/pull/7453/), I followed a pattern where I used ("relative path to the package" +  "operation name"), which I found useful since I could identify where the method was getting invoked specially when we had the same method_name across multiple packages. Not saying this is what we need to do, but as long as we agreed on a common naming convention I am okay. Would like to hear your thoughts.

Reply:
I think ("relative path to the package" + "operation name") is no problem, I will modify it to unify.